pkg/k8s/client.go

```diff
@@ -65,35 +58,18 @@ func newClient(kubeClient kubernetes.Interface, policyClient policyv1alpha1Clien
 		informerInitHandlerMap[informer]()
 	}
 
-	if err := c.run(stop); err != nil {
-		log.Error().Err(err).Msg("Could not start Kubernetes Namespaces client")
-		return nil, err
-	}
-
-	return c, nil
+	return c
 }
 
 // Initializes Namespace monitoring
 func (c *client) initNamespaceMonitor() {
-	monitorNamespaceLabel := map[string]string{constants.OSMKubeResourceMonitorAnnotation: c.meshName}
-
-	labelSelector := fields.SelectorFromSet(monitorNamespaceLabel).String()
-	option := informers.WithTweakListOptions(func(opt *metav1.ListOptions) {
-		opt.LabelSelector = labelSelector
-	})
-
-	informerFactory := informers.NewSharedInformerFactoryWithOptions(c.kubeClient, DefaultKubeEventResyncInterval, option)
-
-	// Add informer
-	c.informers[Namespaces] = informerFactory.Core().V1().Namespaces().Informer()
-
 	// Add event handler to informer
 	nsEventTypes := EventTypes{
 		Add:    announcements.NamespaceAdded,
 		Update: announcements.NamespaceUpdated,
 		Delete: announcements.NamespaceDeleted,
 	}
-	c.informers[Namespaces].AddEventHandler(GetEventHandlerFuncs(nil, nsEventTypes, c.msgBroker))
+	c.informers.AddEventHandler(osminformers.InformerKeyNamespace, GetEventHandlerFuncs(nil, nsEventTypes, c.msgBroker))
```

Comment:
nit: we can drop the prefix `InformerKey` from the variable name. Same applies to other informer keys.
```suggestion
	c.informers.AddEventHandler(osminformers.NamespaceKey, GetEventHandlerFuncs(nil, nsEventTypes, c.msgBroker))
```

Reply:
This prefix is actually a naming convention used in [client-go](https://pkg.go.dev/k8s.io/api/core/v1#NodeSelectorOperator) and other packages when creating variables for a type alias. It helps remove ambiguity when searching for ambiguous or overloaded vars (like namespace or pod)